firestore-multimodal-genai/extension.yaml

```diff
@@ -63,6 +63,17 @@ roles:
       Allows this extension to access the Gemini family of genai models via
       Vertex AI if this provider is chosen.
 
+  - role: monitoring.metricWriter
```

Comment:
make sure to update the version in the extension.yaml and also update the CHANGELOG.md

(make sure to mention in the changelog that we've cleaned up the tests)